internal/cypress/sauce/runner.go

```diff
@@ -24,42 +27,47 @@ type Runner struct {
 	JobStarter      job.Starter
 	JobReader       job.Reader
 	Concurrency     int
+	URL             string
 }
 
 type result struct {
 	suiteName string
+	browser   string
+	job       job.Job
 	err       error
 }
 
 // RunProject runs the tests defined in cypress.Project.
 func (r *Runner) RunProject() (int, error) {
 	log.Error().Msg("Caution: Not yet implemented.") // TODO remove debug
+	exitCode := 1
 
 	// Archive the project files.
 	tempDir, err := ioutil.TempDir(os.TempDir(), "saucectl-app-payload")
 	if err != nil {
-		return 1, err
+		return exitCode, err
 	}
 	defer os.RemoveAll(tempDir)
 
 	zipName, err := r.archiveProject(tempDir)
 	if err != nil {
-		return 1, err
+		return exitCode, err
 	}
 
 	fileID, err := r.uploadProject(zipName)
 	if err != nil {
-		return 1, err
+		return exitCode, err
 	}
 
-	errCount := r.runSuites(fileID)
+	isPassed := r.runSuites(fileID)
```

Comment:
@valentyn88 you can use just `passed` directly here. Or `didPassed`.

Reply:
I fixed it in my PR.